docs/cel2/whats-changed.md

```diff
@@ -0,0 +1,135 @@
+
+See [here](https://specs.Celo.org/l2_migration.html#changes-for-contracts-developers) for the list of hardforks that will be enabled in the first block of the L2.
+
+## Precompiled contracts
+
+All Celo specific precompiles removed except for the transfer precompile which supports Celo token duality (the native asset Celo is also an ERC20 token)
+
+## Randomness
+
+The random contract removed, if randomness is needed then the PREVRANDAO opcode can be used, more details [here](https://specs.Celo.org/l2_migration.html#deactivated-random-contract).
+
+## Blocks
+
+* Block interval changed from 5s to 1s
+* Block gas changed from 50m to 30m
+
+:::note
+Note this results in a 300% increase in gas per second due to the shortened block time
+:::
+
+### Added fields
+* __withdrawals__ & __withdrawalsRoot__ - these fields are inherited from ethereum but not used by the op-stack or Celo, withdrawals will always be an empty list, withdrawalsRoot will always be the empty withdrawals root (`0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421`)
+* __blobGasUsed__ & __excessBlobGas__ - these fields are also inherited from ethereum but not used by the op-stack or Celo, they will always be zero.
+* __parentBeaconBlockRoot__ - set to the parentBeaconRoot of the L1 origin block.
+
+### Removed fields
+* __randomness__ - not needed since the [randomness](#randomness) feature is being removed
+* __epochSnarkData__ - not needed since there will be no [plumo](https://docs.Celo.org/protocol/plumo) support in the Celo L2.
+* __extraData__ - will have the BLS aggregated signature removed, this simplifies the L2 implementation and since we already trust those blocks and distributed rewards for them, the BLS signature is no longer required.
+
+## EIP1559 implementation
+
+Previously our implementation used a smart contract [(here)](https://github.com/celo-org/celo-monorepo/blob/faca88f6a48cc7c8e6104393e49ddf7c2d7d20e3/packages/protocol/contracts-0.8/common/GasPriceMinimum.sol#L162) to calculate the base fee which allowed for governable parameters. Now we use the the standard EIP1559 algorithm with the parameter values being defined in the chain config.
+
+The parameters we are using are:
+
+- Alfajores
+  - EIP-1559 elasticity multiplier: 5
+  - EIP-1559 denominator: 400
+  - EIP-1559 floor: 25 Gwei (Note that the floor is not part of the original EIP-1559 specification, but it did exist in the Celo L1 smart contract implementation)
+
+## RPC API
+
+### Pre-transition data
+
+Old blocks, transactions, receipts and logs will still be accessible via the RPC API but they will differ a bit from the corresponding objects retrieved from the L1 RPC API.
+
+In general the changes involve additional extra unset fields that have been added upstream but were not present on historical Celo L1 objects, and the removal of some unnecessarily set fields on Celo L1 objects.
+
+For in depth details of what is changed see here - (TODO add section in specs covering this)
```

Comment:
What to do with the TODO?

Reply:
Point it to the relevant specs section when it's done - https://github.com/celo-org/celo-blockchain-planning/issues/641